_data/simple-icons.json

```
@@ -2145,6 +2145,11 @@
             "hex": "F56C2D",
             "source": "https://www.origin.com/gbr/en-us/store"
         },
+        {
+            "title": "OSI",
```

I would say the title should be "Open Source Initiative", any particular reason you choose "OSI" instead?

```suggestion
            "title": "Open Source Initiative",
```